client.js

```diff
@@ -0,0 +1,75 @@
+//backbone MVP Model View Presenter
+
+var rcCollection = Backbone.Model.extend({
+  initialize: function (){ //native property no value until valued
+    console.log ('rcCollection created');
+  },
+  });
+ rcCollection.on('change', function(){
```

Comment:
rcCollection is a constructor function at this point, and you haven't created an instance of this collection yet to be able to do things like this. Also, when defining constructors, it standard to capitalize them to make it clear they're constructors, e.g. `RcCollection`. Then, to do what you're trying to do here, you would instantiate a collection of this type (and instances are then lowercase):

`var rcCollection = new RcCollection();`
`rcCollection.on('change', function () {
   console.log('something changed in this collection');
});`
`rcCollection.on('add', function () {
   console.log('something added to this collection');
});`